<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveAs</class>
 <widget class="QDialog" name="SaveAs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>194</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>291</width>
    <height>194</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>291</width>
    <height>194</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: rgb(191,193,194);
}

/*	Push button*/
QPushButton {
	min-height: 19px;
	border: 1px solid gray;
	border-radius: 3px;
	background-color: lightgray;
	selection-background-color: gray;
}

QPushButton:hover {
	background-color: gray;
	border-radius: 3px;
}


/*Tool button*/
QToolButton {
	max-width: 20px;
	max-height: 20px;
	border: 0px solid gray;
}

/*Line edit*/
QLineEdit {
    border: 1px solid gray;
	border-radius: 3px;
	height: 19px;
    background-color: lightgray;
    selection-background-color: darkgray;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="m_pFrameTitle">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>43</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>43</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(105,105,105);
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="m_pLabelTitle">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Save As</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_pToolButtonCancel">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../../Resources/Icons/close-icon.png</normaloff>../../../../Resources/Icons/close-icon.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="m_pLayoutData">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="horizontalSpacing">
      <number>10</number>
     </property>
     <property name="verticalSpacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="m_pLabelCategory">
       <property name="text">
        <string>File:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="m_pLabelProduct">
       <property name="text">
        <string>Directory:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="m_pLineEditFileName">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>19</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>19</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <widget class="QLineEdit" name="m_pLineEditDirectory">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>19</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>19</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_pPushButtonBrowseDir">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>19</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>19</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="text">
          <string>Browse</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0" colspan="2">
      <layout class="QHBoxLayout" name="m_pLayoutButtonBox">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QPushButton" name="m_pPushButtonApply">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_pPushButtonCancel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="2" column="0" colspan="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
